source/index.html.md.erb

```diff
@@ -44,24 +44,23 @@ Codes appear in three contexts:
 
 ## How do I connect to this API?
 
-### Authentication and authorisation
+> While the API is in alpha, you can get credentials by [contacting](/get-help)
```

Comment:
`[contacting the Apply service team]` would be better! Minor.

Reply:
Thank you – fixed